pom.xml

```diff
@@ -1961,6 +1966,10 @@
                         <includes>
                             <include>**/*Test.java</include>
                         </includes>
+                        <excludes>
+                            <exclude>**/*IT.java</exclude>
+                            <exclude>**/*IT.scala</exclude>
```

Comment:
aren't these kind of pointless? it woldn't match *Test.java if it ends with IT.java...